src/Glimpse.Common/Broker/Message.cs

```diff
@@ -9,14 +9,16 @@ public class Message : IMessage
     {
         public Guid Id { get; set; }
 
-        public string Type { get; set; }
+        public IEnumerable<string> Types { get; set; }
 
+        [JsonConverter(typeof(RawStringConverter))]
         public string Payload { get; set; }
 
-        public MessageContext Context { get; set; }
+        public int Ordinal { get; set; }
 
-        public IEnumerable<string> Tags { get; set; }
+        public MessageContext Context { get; set; }
 
+        [JsonIgnore]
```

Comment:
Yep thats what I mean. SignalR does it for their message envelope which made me think of it. Not sure what the actual difference would be. Maybe a good jump-in task to try it out and test.